modules/ROOT/pages/index.adoc

```diff
@@ -1,39 +1,205 @@
-= About Mule Runtime 4.x
+= Mule Runtime Overview
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: mule, getting started, transform, message, payload
 
-// This guide offers information about how to use Mule Runtime, commonly called Mule, or Mule ESB to integrate systems, orchestrate web services and configure the runtime environment.
+Mule Runtime (or Mule) is a lightweight integration engine that runs Mule apps.
+Rather than creating multiple point-to-point integrations between systems,
+services, APIs, and devices, Mule apps manage message routing, data mapping,
+orchestration, reliability, security, and scalability between nodes.
 
-Mule is a lightweight integration engine that runs Mule apps. Rather than creating multiple point-to-point integrations between systems, services, APIs, and devices, you use Mule to create apps that manage message routing, data mapping, orchestration, reliability, security, and scalability between nodes.
+== Event-Driven Architecture
 
-The runtime defines the rules for how event data is handled in an application.  Mule apps receive events, then process and route them to other endpoints.
+Mule works by responding to events that are initiated by external or internal
+resources. Mule supports a variety of listeners, as well as internally
```

Comment:
Shouldn't we say `sources` instead of `listeners`?

Reply:
done